codespell_lib/data/dictionary.txt

```diff
@@ -7561,13 +7561,18 @@ fertily->fertility
 fetaure->feature
 fetaures->features
 feture->feature
+fewg->fugue
```

Comment:
Suggest few

Reply:
Done

Reply:
Thanks, I suspect few is more likely as a word.

Reply:
Done